joomla/joomla-cms
# [com_users] debuguser & debuggroup: use the same tree level visual

#### Summary of Changes

This PR does for users debug & user groups debug, the same visual change in the tree structure as done:
- for categories https://github.com/joomla/joomla-cms/pull/10213
- for tags https://github.com/joomla/joomla-cms/pull/10526
- for user group https://github.com/joomla/joomla-cms/pull/10528

After PR (example)
![image](https://cloud.githubusercontent.com/assets/9630530/15325759/52e89e34-1c43-11e6-8550-5997a9f89a95.png)
#### Testing Instructions
1. Use latest staging
2. Apply patch https://github.com/joomla/joomla-cms/pull/10528 (to have the layout) and then this patch
3. Go to Global config and enable System Debug
4. Go to Users -> Manage, click "Debug Permission report" on any user and check the tree structure visual
5. Go to Users -> Groups, click "Debug Permission report" on any user group and check the tree structure visual
